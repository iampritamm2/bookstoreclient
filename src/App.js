import React from "react";


const App = () =>{
    return <div>my first componemt</div>;
}

export default App;