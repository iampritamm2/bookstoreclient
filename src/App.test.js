import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App component",()=>{
    it("should rendur app with error",() => {
        const { asFragment } = render(<App />);
        expect(asFragment()).toMatchSnapshot();
    });
});