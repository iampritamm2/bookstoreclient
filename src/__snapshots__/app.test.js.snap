// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should rendur app with error 1`] = `
<DocumentFragment>
  <div>
    my first componemt
  </div>
</DocumentFragment>
`;
